### PR TITLE
#1028 organization model and serializers to support event association…

### PR DESCRIPTION
**Pull request by to-sta**
…s and update admin display

### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch
- [x] I have run the tests for the backend and frontend depending on what's needed for my changes as described in the [testing section of the contributing guide](CONTRIBUTING.md#testing)

---

### Description

I worked with @momanyisamuel on this today. There is still quite some work that needs to be done here. The organization api, get the org_text now and we started working on org_events.

### Related issue

- #1028

### CODE REVIEW

**Comment by andrewtavis** [comment]
Really nice, @to-sta and @momanyisamuel :) Seeing the responses from organizations for the first time just now 😊

**Comment by andrewtavis** [comment]
Ok so post 58ed272 the `/organizations` route and organization About pages load in :) Still some work to do, but it's sooo much easier now! Thank you both so much!

**Comment by andrewtavis** [comment]
I'm also not sure on the i18n pre-commit errors/warnings 🤔 @mattburnett-repo was getting the same as @to-sta, I believe, but then when I took them out they all ran fine for me. Let's keep an eye on this!

File: frontend/stores/organization.ts [hunk 1]
```diff
@@ -96,96 +92,32 @@ export const useOrganizationStore = defineStore("organization", {
...
-          //       {}
-          //     )
-          // ),
-        ]);
+      const [responseOrg] = await Promise.all([
+        useAsyncData(
+          async () =>
+            await fetchWithoutToken(`/entities/organizations/${id}`, {})
+        ),
+      ]);
```
Comment (cquinn540) [inline frontend/stores/organization.ts:100]: `Promise.all()` is no longer necessary since there is only one async request

Comment (andrewtavis) [reply frontend/stores/organization.ts:100]: Thanks for pointing this out, @cquinn540!

**Review by andrewtavis — APPROVED** [review]
Alright :) So with this we're bringing all of the various data for orgs, groups and events in one call each. Thanks so much for the initial work here, @to-sta and @momanyisamuel! 🙌 Really is making things so much easier to work with. Thanks also for the review, @cquinn540!

Current state isn't perfect, but is good enough to bring in. The event page is loading the location from the backend, but the navigation is a bit buggy and some of the page elements aren't perfect (map marker color for some reason isn't reactive).

I'll look into this more this weekend! Would be great if people had time to check things a bit :)

**Pull request merged**